docs/modules/ROOT/pages/cops_performance.adoc

```diff
@@ -1269,7 +1269,7 @@ This cop identifies places where `map { ... }.compact` can be replaced by `filte
 [#safety-performancemapcompact]
 === Safety
 
-This cop's autocorrection is unsafe because `map { ... }.compact` might yield
+This cop is unsafe because `map { ... }.compact` might yield
```

Comment:
This `docs/modules/ROOT/pages/cops_performance.adoc` is auto-generated. Can you update the source code document instead? Also, no update to `docs/modules/ROOT/pages/cops_performance.adoc` is needed.